decidim-blogs/app/views/decidim/blogs/posts/index.html.erb

```diff
@@ -1,10 +1,35 @@
 <%= render partial: "decidim/shared/component_announcement" %>
 
-<div class="row column">
-  <div id="blogs" class="columns medium-7 mediumlarge-8">
-    <%= render partial: "posts" %>
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_blogs", media: "all" %>
+<% end %>
+
+<% content_for :aside do %>
+  <h1 class="h1 decorator"><%= t("decidim.components.blogs.name") %></h1>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_two_col" do %>
+
+  <%# REDESIGN_PENDING: sorting component, it requires Turbo, perhaps a cell or component %>
+  <div class="text-right mb-8">
+    <div class="inline-flex items-center">
+      <label for="filter-1" class="text-sm text-secondary px-2 py-1 cursor-pointer">
+        <input type="radio" name="filter" id="filter-1" class="appearance-none peer" checked>
+        <span class="border-b-2 border-transparent peer-checked:border-secondary peer-checked:font-semibold hover:border-secondary transition">Aleatori</span>
```

Comment:
It's a mock. Take a look to the REDESIGN_PENDING comment previous to this block. When you come across these comments, keep in mind something is not properly implemented.

The reason of such naming is about to avoid that the tests fail (looking for TODO comment). We'll come back to them in a future.